daemon/container.go

```diff
@@ -353,9 +359,14 @@ func (container *Container) cleanup() {
 		container.daemon.unregisterExecCommand(eConfig)
 	}
 
-	container.UnmountVolumes(false)
+	container.unmountVolumes(false)
 }
 
+// KillSig sends the container the given signal. This wrapper for the
+// host specific kill command prepares the container before attempting
+// to send the signal. An error is returned if the container is paused
+// or not running, or if there is a problem returned from the
+// underlying kill command.
 func (container *Container) KillSig(sig int) error {
```

Comment:
It can become `Signal` in that the `kill` command is used to send signals. Or it can become `Kill` and `Kill` can become `KillForcefully` or some such that indicates it's unceremonious nature.

I would want a second and third opinion before making this change.